src/controllers/monster.js

```diff
@@ -885,6 +888,114 @@ class Monster extends Controller {
 						}
 
 						const templateType = (data.iv === -1) ? 'monsterNoIv' : 'monster'
+						
+						sendscout: if (data.iv === -1 && this.config.general.scoutURL) {
+
+						    const coords = JSON.stringify([[parseFloat(data.latitude.toFixed(5)),parseFloat(data.longitude.toFixed(5))]])
+							const headers = {
+									'Content-Type': 'application/json',
+									'X-Dragonite-Secret': this.config.general.dragoniteSecret
+									}
+							const url = this.config.general.scoutURL
+							
+							// Try avoid sending dupes, check current against last used:
+							if (typeof lastcoords === 'undefined' )  {
+								   const lastcoords = coords	// lastcoords not yet defined, set value
+								} else if (lastcoords === coords) {
+								   this.log.info(`[SCOUT] Skipping duplicate attempt with ${coords}`)
+								   break sendscout; 
```

Comment:
I think this will break from the wrong place; instead you should be setting a flag and checking it later.

However, there is a broader problem here in that this code runs on <n> threads, and so whatever lastcoords is is only for this thread and so this duplicate elimination is unlikely to succeed in this form

Reply:
In my googling, labeling and breaking out of a conditional is acceptable.  (and it does seem to work).  
my ref:   <https://stackoverflow.com/questions/4851657/call-break-in-nested-if-statements>

Reply:
The 'broader problem' of determining how to make lastcords (or any varable) persist outside of the thread is where I am stuck and can use more hints.

Reply:
Please keep in mind, my programming experiences is near zero here and so I'm learning as I go.

Reply:
How to prevent duplicates in a multi-threaded?  In searching I came up with using a lock file somehow or adding a redis module and using redis to keep state variables -- seems not quite suitable?   I can write the coords to the database in a new table and check there?

Reply:
My duplicate noIV hits are extremely low, and so the amount of effort to solve preventing dupes I think is beyond me...